### PR TITLE
Update Russian translation

### PR DESCRIPTION
**Pull request by dmitry-pshenitsin**

### CODE REVIEW

**Review by undef21 — CHANGES_REQUESTED** [review]
Hello @dmitry-pshenitsin. I left several comments.

**Comment by ihhub** [comment]
Marking this pull request as WIP since the author has no been updating it for a long time.

File: files/lang/ru.po [hunk 1]
```diff
@@ -1109,7 +1109,7 @@ msgid "Conquer and Unify"
 msgstr "Завоевать и объединить"
 
 msgid "Crazy Uncle Ivan"
-msgstr "Безумный дядя"
+msgstr "Безумный дядя Ваня"
```
Comment (a1exsh) [inline files/lang/ru.po:1112]: I think that didn't fit in the UI.

**Comment by ihhub** [comment]
I am closing this pull request as stalled.

**Pull request closed without merging**